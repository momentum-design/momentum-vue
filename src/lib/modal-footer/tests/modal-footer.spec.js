import { mount } from '@vue/test-utils';
import ModalFooter from '../index.vue';

describe('ModalFooter', () => {
  it('should match snapshot', () => {
    const wrapper = mount(ModalFooter);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render one ModalFooter', () => {
    const wrapper = mount(ModalFooter);

    expect(wrapper.findAll('.md-modal__footer').length).toEqual(1);
  });

  it('should render children', () => {
    const wrapper = mount(ModalFooter, {
      slots: {
        default: '<div class="testingforMF" />'
      }
    });

    expect(wrapper.findAll('.testingforMF').length).toEqual(1);
  });
});

