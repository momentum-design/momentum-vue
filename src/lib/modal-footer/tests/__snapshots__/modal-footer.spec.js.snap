// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter should match snapshot 1`] = `<div class="md-modal__footer"></div>`;
